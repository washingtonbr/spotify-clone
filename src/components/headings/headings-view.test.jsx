import React from 'react';
import renderer from 'react-test-renderer';
import HeadingsView from './headings-view';

describe('HeadingsView', () => {
  test('renders correctly', () => {
    const tree = renderer
      .create(<HeadingsView>Title</HeadingsView>)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
