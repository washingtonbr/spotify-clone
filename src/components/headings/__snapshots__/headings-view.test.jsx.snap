// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeadingsView renders correctly 1`] = `
<h1
  className="sc-AxjAm dwydzm"
>
  Title
</h1>
`;
